plugin-dev/Source/Sentry/Private/Android/Java/SentryBridgeJava.java

```diff
@@ -32,6 +35,15 @@ public void configure(SentryAndroidOptions options) {
 				options.setEnvironment(environment);
 			}
 		});
+
```

Comment:
Do we need to check for `settings->EnableAutoLogAttachment` beforehand?

Reply:
Before calling the Java bridging code, this check is performed in `SentrySubsystemAndroid::InitWithSettings`. If the `EnableAutoLogAttachment` option is enabled we'll pass a proper log file path here and configure the global scope accordingly. Otherwise, the path string will be empty and the scope configuration will be skipped.